ui/src/app/applications/components/utils.scss

```diff
@@ -14,6 +14,10 @@
     }
 }
 
+.propagation-policy-option :hover {
+    cursor: pointer;
+}
```

Comment:
Can you target it to just the `input` and `label` elements? I think this will show the pointer in the padding to the right of the label, which is not clickable.

Reply:
sure